hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -529,22 +529,17 @@ private void checkIterationMoveResults() {
       allFuturesResult.get(config.getMoveTimeout().toMillis(),
           TimeUnit.MILLISECONDS);
     } catch (InterruptedException e) {
+      long cancelCount = cancelAndCountPendingMoves();
+      metrics.incrementNumContainerMovesFailed(cancelCount);
+      LOG.warn("Container balancer is interrupted and moves are cancelled {}",
+          cancelCount);
```

Comment:
We do not need to handle this since InterruptedException would occur in case balancer is stopped. We shouldn't mark the moves as failed then. It could be misleading.

Reply:
Removed the case of counting as failed. Just cancelling the moves. But as cancel, possibly of exception caught in whenComplete(), that will be counted.